homeassistant/components/ozw/climate.py

```diff
@@ -154,6 +155,13 @@ def async_add_climate(values):
     )
 
 
+def convert_units(units):
```

Comment:
To test this, since it's an isolated helper, we can test it separately. Just call the function in the test with a unit argument and assert that the expected value is returned.